fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/postgresql/PostgreSqlAdapter.java

```diff
@@ -0,0 +1,266 @@
+    public void createTenantPartitions(Collection<Table> tables, String schemaName, int newTenantId, int extentSizeKB) {
+        warnOnce(MessageKey.PARTITIONING, "PostgreSql does not support tenant partitioning");
+    }
+
+    @Override
+    public void createRowType(String schemaName, String typeName, List<ColumnBase> columns) {
+        warnOnce(MessageKey.ROW_TYPE, "Create row type not supported in PostgreSql");
+    }
+
+    @Override
+    public void createArrType(String schemaName, String typeName, String valueType, int arraySize) {
+        warnOnce(MessageKey.ROW_ARR_TYPE, "Create array row type not supported in PostgreSql");
+    }
+
+    @Override
+    public void dropType(String schemaName, String typeName) {
+        warnOnce(MessageKey.DROP_TYPE, "Drop type not supported in PostgreSql");
+    }
+
+    @Override
+    public void createOrReplaceProcedure(String schemaName, String procedureName, Supplier<String> supplier) {
+        warnOnce(MessageKey.CREATE_PROC, "Create procedure not supported in PostgreSql");
+    }
+
+    @Override
+    public void dropProcedure(String schemaName, String procedureName) {
+        warnOnce(MessageKey.DROP_PROC, "Drop procedure not supported in PostgreSql");
+    }
+
+    @Override
+    public void createTablespace(String tablespaceName) {
+        warnOnce(MessageKey.TABLESPACE, "Create tablespace not supported in PostgreSql");
+    }
+
+    @Override
+    public void dropTablespace(String tablespaceName) {
+        warnOnce(MessageKey.TABLESPACE, "Drop tablespace not supported in PostgreSql");
+    }
+
+    @Override
+    public void detachPartition(String schemaName, String tableName, String partitionName, String newTableName) {
+        warnOnce(MessageKey.PARTITIONING, "Detach partition not supported in PostgreSql");
+    }
+
+    @Override
+    public void removeTenantPartitions(Collection<Table> tables, String schemaName, int tenantId,
+            String tenantStagingTable) {
+        warnOnce(MessageKey.PARTITIONING, "Remove tenant partitions not supported in PostgreSql");
+    }
+
+    @Override
+    public void createTablespace(String tablespaceName, int extentSizeKB) {
+        warnOnce(MessageKey.TABLESPACE, "Create tablespace not supported in PostgreSql");
+    }
+
+    @Override
+    public boolean doesTableExist(String schemaName, String tableName) {
+        PostgreSqlDoesTableExist dao = new PostgreSqlDoesTableExist(schemaName, tableName);
+        return runStatement(dao);
+    }
+
+    @Override
+    public void createSequence(String schemaName, String sequenceName, int cache) {
+        /* CREATE SEQUENCE fhir_sequence
+         *     AS BIGINT
+         *     START WITH 1
+         *     CACHE 1000
+         *     NO CYCLE;
+        */
+        // PostgreSql doesn't support CACHE
+        final String sname = DataDefinitionUtil.getQualifiedName(schemaName, sequenceName);
+        final String ddl = "CREATE SEQUENCE " + sname + " AS BIGINT START WITH 1 NO CYCLE";
```

Comment:
start at 1000 always.  it give space to use the ID with specific mappings. for instance RESOURCE_TYPE are hard coded, and we don't want it to client.

Reply:
done